web/src/containers/activity/CostAllocateFFPQuarterly.js

```diff
@@ -27,8 +28,10 @@ class CostAllocateFFPQuarterly extends Component {
         [year]: { [q]: { [name]: +e.target.value } }
       }
     };
-    const { aKey, update } = this.props;
+    const { aKey, update, announce, quarterlyFFP } = this.props;
     update(aKey, change, true);
+    // ARIA region update goes in here somewhere
+    announce(quarterlyFFP[year][q][name].dollars);
```

Comment:
Either I'm calling the old props, or I'm expecting this to be synchronous when it's async and I need to figure out how not to do that.

Reply:
Thanks so much for the thoughtful response! I actually started down the `Promise` path and felt icky about it, so I'm glad that that was a reasonable gut instinct. 😄 

I like the approach of #2, but I'm also not the hugest fan of making a specific action to deal with individual components' structure because ultimately the ARIA region only cares about a string. I _think_ I figured out how to deal with this in `componentDidUpdate` with generic actions that can be reused anywhere we might need them. Will push it up so we can chat Monday!